lib/mongodb.js

```diff
@@ -732,11 +732,28 @@ MongoDB.prototype.buildSort = function(model, order) {
   return sort;
 };
 
+var FEET2METER = 0.3048;
+
+function convertToMeter(distance, unit) {
+  switch (unit) {
+    case 'meters':
+      return distance;
+    case 'kilometers':
+      return distance * 1000;
+    case 'miles':
+      return distance * 1600;
+    case 'feet':
+      return distance * FEET2METER;
```

Comment:
just put 0.3048 directly in here since your `case 'feet'` already implies the conversion unit.